docs/configuration/config-file-reference.md

```diff
@@ -4546,4 +4546,8 @@ otel:
     # Skip validating server certificate.
     # CLI flag: -tracing.otel.tls.tls-insecure-skip-verify
     [tls_insecure_skip_verify: <boolean> | default = false]
+
+  # If the configured key is specified, Otel sampler will sample the trace.
+  # CLI flag: -tracing.otel.force-tracing-key
+  [force_tracing_key: <string> | default = ""]
```

Comment:
The sampler will check whether the configured tracing key is specified in the trace attribute, if it is, it samples the trace.
This requires some additional works on our HTTP middleware. The middleware should be able to check whether an HTTP header is present, if it is, set additional attribute when starting a new span. Now, we cannot do it easily because of the weaveworks common library we are using.